packages/mobile/ios/Quiet.xcodeproj/project.pbxproj

```diff
@@ -5011,24 +5020,6 @@
 			shellPath = /bin/sh;
 			shellScript = "#!/bin/sh\nset -ex\nNODEJS_ASSETS_DIR=\"$( cd \"$PROJECT_DIR\" && cd ../nodejs-assets/ && pwd )\"\nNODEJS_BUILT_IN_MODULES_DIR=\"$( cd \"$PROJECT_DIR\" && cd ../nodejs-modules/ && pwd )\"\nif [ -d \"$CODESIGNING_FOLDER_PATH/nodejs-project/\" ]\nthen\nrm -rf \"$CODESIGNING_FOLDER_PATH/nodejs-project/\"\nfi\nif [ -d \"$CODESIGNING_FOLDER_PATH/builtin_modules/\" ]\nthen\nrm -rf \"$CODESIGNING_FOLDER_PATH/builtin_modules/\"\nfi\nrsync -av --delete \"$NODEJS_ASSETS_DIR/nodejs-project\" \"$CODESIGNING_FOLDER_PATH\"\nrsync -av --delete \"$NODEJS_BUILT_IN_MODULES_DIR/builtin_modules\" \"$CODESIGNING_FOLDER_PATH\"\n";
 		};
-		18D742A42A41DAD8007D4C4E /* Remove Simulator Strips */ = {
```

Comment:
this build phase would remove the new nodejs executable from the `NodejsMobile.framework` selected from `NodejsMobile.xcframework` and *nothing* else. It's safe to delete and crucial to actually getting the proper executable installed on your device/simulator